WebPhapp/backend/block_helper.js

```diff
@@ -169,15 +169,133 @@ module.exports = {
         );
         
         // Submitting prescription transaction.
-        let encoded_transaction = transaction.encodeABI();
+        let encodedTransaction = transaction.encodeABI();
         let block = await blockchain.web3.eth.sendTransaction({
-            data: encoded_transaction,
+            data: encodedTransaction,
             from: blockchain.account,
             to: blockchain.patient.options.address,
             gas: 50000000
         });
         
         // Return Transaction object containing transaction hash and other data
         return block;
+    },
+
+    /*
+    This function redeems a prescription on the blockchain,
+	decrementing refills left, and adding a new date to fulfillmentDates
+    Args:
+        chainIndex (int)
+        date (int)
+    Returns:
+        Transaction object
+    Example:
+        redeem(0, 123456543)
+    */
+    redeem: async function(chainIndex, date) {
```

Comment:
At what point are the dates converted back to integers from strings on the frontend? @jacobkrantz

Reply:
`convertDatesToString()` in `server.js` should always do this.